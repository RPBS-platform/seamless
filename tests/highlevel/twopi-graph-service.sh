rm -f twopi-colored-service.seamless
###docker run --rm -v /tmp/redis:/data redis bash -c 'rm -rf /data/*'
###redisID=`docker run --rm -p 6379:6379 --name redis-container -v /tmp/redis:/data -d redis redis-server --appendonly yes`

python3 simple-pi-redis.py
python3 ~/seamless-scripts/color-graph-service.py 8 &
sleep 3
python3 ~/seamless-scripts/jobslave.py --time 5 --communion_incoming localhost:8600 &
sleep 3
python3 twopi-graph-service-submit.py twopi.seamless twopi-colored-service.seamless
wait
md5sum twopi-colored-service.seamless

###docker stop $redisID