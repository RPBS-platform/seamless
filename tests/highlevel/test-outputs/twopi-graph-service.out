c28719736c80dcc1f867100c98a6651f  twopi.seamless
c28719736c80dcc1f867100c98a6651f  /tmp/twopi-redis.seamless
TaskManager, tasks: 11 undestroyed
Set up a communion outgoing port 8602
INCOMING jobslave jobmaster-service
ADD SERVANT buffer
ADD SERVANT buffer_length
ADD SERVANT semantic_to_syntactic
Set up a communion outgoing port 8600
OUTGOING jobmaster-service jobslave
ADD SERVANT buffer
ADD SERVANT buffer_length
ADD SERVANT semantic_to_syntactic
ADD SERVANT transformation
REMOVE SERVANT buffer
REMOVE SERVANT buffer_length
REMOVE SERVANT semantic_to_syntactic
REMOVE SERVANT transformation
b4da0d18c639d022e943156c448a02ba  twopi-colored-service.seamless
